### ISSUE TITLE
language used?

### ISSUE THREAD

**Issue by hugoledoux** [cityjson/metadata-extended#5]
shouldn't this added?

**Comment by jliempt** [comment]
As array, for datasets from multilingual countries? Or is it generally better to stimulate separate files per language?

**Comment by hugoledoux** [comment]
no, to specify what language is the dataset using (for the attributes I assume), for the 3dbag.nl it would be 'dut' (https://geonovum.github.io/Metadata-ISO19115/#codelist-iso639-2, I don,t know a good page about iso19115 in english but that one is great)

although, I see `"dak_type": "single horizontal",` and `"reconstruction_skipped": false` ...
